lib/linter-plus.coffee

```diff
@@ -79,21 +73,37 @@ class Linter
   getLinters: ->
     @linters
 
+  setMessages: (linter, messages) ->
+    @messages.set(linter, Helpers.validateResults(messages))
+    @emitter.emit 'did-change-messages', @messages
+    @views.render()
+
+  deleteMessages: (linter, messages) ->
+    @messages.delete(linter)
+    @emitter.emit 'did-change-messages', @messages
+    @views.render()
+
+  getMessages: ->
+    return @messages
+
+  onDidChangeMessages: ->
+    return @emitter.on 'did-change-messages', callback
+
   onDidChangeProjectMessages: (callback) ->
-    @emitter.on 'did-change-project-messages', callback
+    console.warn("Linter::onDidChangeProjectMessages is deprecated, use Linter::onDidChangeMessages instead")
+    return @onDidChangeMessages(callback)
 
   getProjectMessages: ->
-    @messagesProject
+    console.warn("Linter::getProjectMessages is deprecated, use Linter::getMessages instead")
+    return @getMessages()
 
   setProjectMessages: (linter, messages) ->
-    @messagesProject.set(linter, Helpers.validateResults(messages))
-    @emitter.emit 'did-change-project-messages', @messagesProject
-    @views.render()
+    console.warn("Linter::setProjectMessages is deprecated, use Linter::setMessages instead")
+    return @setMessages(linter, messages)
 
   deleteProjectMessages: (linter) ->
-    @messagesProject.delete(linter)
-    @emitter.emit 'did-change-project-messages', @messagesProject
-    @views.render()
+    console.warn("Linter::deleteProjectMessages is deprecated, use Linter::deleteMessages instead")
```

Comment:
I was gonna switch to that before merge, these warns were temp. But still thanks for the link :clap: